### PR TITLE
[Blitz] Added docs for run_condition actions  if, elif, else

### PR DESCRIPTION
**Pull request by lsheikal**

### CODE REVIEW

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 1]
```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
...
+            pkg: genie.libs.sdk
+            class: triggers.blitz.blitz.Blitz
+        devices: ['uut']
+        test_sections:
+            - apply_config:
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output                 # the 1500 is stored in gims_output
```
Comment (tahigash) [inline docs/blitz/design/advanced_actions/advanced.rst:586]: `gims_output`/`gims_output_1` -> `mtu1`/`mtu2` is better

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 2]
```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
...
+                     arguments:
+                       interface: GigabitEthernet1
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output_1                 # the 2500 is stored in gims_output
+                     arguments:
+                       interface: GigabitEthernet10
```
Comment (tahigash) [inline docs/blitz/design/advanced_actions/advanced.rst:595]: I think this api section is not needed in this example?

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 3]
```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
...
+            pkg: genie.libs.sdk
+            class: triggers.blitz.blitz.Blitz
+        devices: ['uut']
+        test_sections:
+            - apply_config:
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output                 # the 1500 is stored in gims_output
```
Comment (tahigash) [inline docs/blitz/design/advanced_actions/advanced.rst:626]: same -> `mtu1`/`mtu2`

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 4]
```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
...
+                     arguments:
+                       interface: GigabitEthernet1
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output_1                 # the 2500 is stored in gims_output
+                     arguments:
+                       interface: GigabitEthernet10
```
Comment (tahigash) [inline docs/blitz/design/advanced_actions/advanced.rst:635]: not used again? please remove or please update example to use this

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 5]
```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
...
+                      - parse:
+                          command: show version
+                          device: uut
+                      - sleep:
+                          sleep_time: 1
+                  - else:                                 # actions under else is executed here
+                    actions:
+                      - parse:
+                          command: show version
+                          device: uut
```
Comment (tahigash) [inline docs/blitz/design/advanced_actions/advanced.rst:608]: not sure why `show version` here. maybe simply use `print` action to tell the result here like `mtu is not 1200`.

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 6]
```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
...
+                  - elif: "%VARIABLES{gims_output} == 1500"  # gims_output output is 1500, elif is executed
+                    actions:
+                      - parse:
+                          command: show version
+                          device: uut
+                  - else:
+                    actions:
+                      - parse:
+                          command: show version
+                          device: uut
```
Comment (tahigash) [inline docs/blitz/design/advanced_actions/advanced.rst:658]: same. `print` action to tell result instead of `show version`

**Review by tahigash — CHANGES_REQUESTED** [review]

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 1 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst:586]: updated

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 2 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst:595]: updated.

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 3 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst:626]: updated.

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 4 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst:635]: updated.

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 5 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst:608]: updated.

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 6 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst:658]: updated.

File: docs/blitz/design/advanced_actions/advanced.rst [hunk 7]
```diff
@@ -389,6 +389,10 @@ To run actions with a conditional statement, *Blitz* expects:
 
 * An if statement with boolean value (True or False statement).
 
+* (Optional) An elif statement with boolean value (True or False statement).
+
+* (Optional) An else statement, the actions under else is executed when if, elif statement fails.
```
Comment (LukasMcClelland) [inline docs/blitz/design/advanced_actions/advanced.rst:394]:
```suggestion
* (Optional) An else statement, the actions under else are executed when the booleans for the if and elif statements are all False.

```

**Review by LukasMcClelland — APPROVED** [review]
Looks good

File: docs/blitz/design/advanced_actions/advanced.rst (hunk 7 above)
Comment (lsheikal) [reply docs/blitz/design/advanced_actions/advanced.rst]: updated.

**Review by tahigash — APPROVED** [review]
thanks for addressing comments. approved

**Pull request merged**